quilt/tools/package.py

```diff
@@ -37,9 +20,9 @@
 
 
 class ParquetLib(Enum):
+    SPARK = 'pyspark'
```

Comment:
I rely on the order later, in `for lib in ParquetLib`; I changed it so `SPARK` is preferred, like before.